pkg/agent/baker.go

```diff
@@ -316,6 +316,9 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"Disable1804SystemdResolved": func() bool {
 			return config.Disable1804SystemdResolved
 		},
+		"EnableUnattendedUpgrade": func() bool {
+			return !config.DisableUnattendedUpgrades
```

Comment:
nit: shall we call it enable to avoid the double negative?

Reply:
default for a bool is false though and for now the default needs to be unattended upgrades enabled.

Reply:
with new refactor at least go rid of the not